README.md

```diff
@@ -1,66 +1,255 @@
 
 ## Getting Started
 
-_This is an example of how you may give instructions on setting up your project locally. To get a local copy up and running follow these simple example steps._
-
-_Listing specific steps helps remove ambiguity and gets people to using your project as quickly as possible._ 
-
 ### Prerequisites
 
-_If your project only runs in a specific context like a particular programming language version or operating system or has dependencies that have to be installed manually, also add a Prerequisites subsection._
+* python3
+* IP connectivity to iPDU controller
 
 ### Installation
 
-### Usage
+1. Create and activate python3 virtual environment
+```bash
+# python3 -m venv ipdu
+# cd ipdu
+# . bin/activate
+```
+2. Clone ServerTech iPDU tool
+```bash
+# git clone git@github.com:Cray-HPE/servertech-pdu.git
 
-_Use examples liberally, and show the expected output if you can. It's helpful to have inline the smallest example of usage that you can demonstrate, while providing links to more sophisticated examples in official documentation if they are too long to reasonably include in the README._
+or
 
-_Depending on what you are making, it can be a good idea to include screenshots or even a video (you'll frequently see GIFs rather than actual videos). Tools like ttygif can help, but check out Asciinema for a more sophisticated method._
+# git clone https://github.com/Cray-HPE/servertech-pdu.git
+```
 
-## Compatibility
-
-_List the versions of the project and their associated dependencies and any other compatibility information. A table is often the easiest way to communicate compatibility information. The [Kubernetes python client](https://github.com/kubernetes-client/python#compatibility) is a good example._
-
-## Support
-
-_Tell people where they can go to for help. It can be any combination of an issue tracker, a chat room, an email address, official documentation, etc._
-
-* [Slack Channel(s)]
-* [Admin or installation guide section]
-* [CSM SIG group discussions in Cray-HPE/community repository]
-
-## Roadmap (optional)
+### Usage
 
-_If you have ideas for releases in the future, it is a good idea to provide links to the roadmap, wherever they reside (CASMFEAT-###)._
+```bash
+# ./servertech_pdu.py --help
+usage: servertech_pdu.py [-h] [--on] [--off] [--reboot] [--status]
+                         [--pdus PDUS] [--outlets OUTLETS] [--groups GROUPS]
+                         [--user USER] [--passwd PASSWD] [--file FILE]
+                         [--version]
+
+Server Tech iPDU control.
+
+optional arguments:
+  -h, --help         show this help message and exit
+  --on               turn selected outlets/groups On
+  --off              turn selected outlets/groups Off
+  --reboot           reboot selected outlets/groups
+  --status           get status of outlet or group
+  --pdus PDUS        iPDU controller hostnames, IPv4 addrs, and/or IPv6 addrs
+  --outlets OUTLETS  target outlets
+  --groups GROUPS    target groups
```

Comment:
Are these groups the same thing as outlet groups configured on the PDU, or are they a concept of this tool? If they are configured on the PDUs what creates these groups?

Reply:
They are the groups on the iPDU themselves. Site personnel are working on creating the groups. In the end, this tool should handle all of it IMO.